src/voters/voterFile/schemas/ScheduleOutreachCampaign.schema.ts

```diff
@@ -12,15 +12,14 @@ export class ScheduleOutreachCampaignSchema extends createZodDto(
         .object(
           CUSTOM_FILTERS.reduce(
             (acc, filterKey) => {
-              acc[filterKey] = z.boolean().optional()
+              acc[filterKey] =
+                filterKey === 'audience_request'
```

Comment:
`enum`?

Reply:
Oh.. it's already right here.. use this: https://github.com/thegoodparty/gp-api/pull/187/files#diff-6e50362ee30dfcdd287568bf0dbe361431bc6e8a2afaa96378d9889e510d5d73R50

Reply:
That is an array, so it get's a little dirty trying to reuse it.